src/NzbDrone.Core/Notifications/Webhook/WebhookPayload.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookPayload
+    {
+        public string EventType { get; internal set; }
+        public WebhookSeries Series { get; internal set; }
+        public WebhookEpisodeFile EpisodeFile { get; internal set; }
+        public WebhookEpisode Episode { get; internal set; }
```

Comment:
@markus101 so what do you think, include everything in the `episode` object? is there a point in having release and episdoe separated?